### PR TITLE
Create skeletal Node apps for the curator and data services.

### PR DESCRIPTION
**Pull request by axmb**
Basic set up for our curator service and central data service.

Some points of note:
- Didn't add many convenience `scripts`. I'll do this later, or leave it for Tim's sprint task. If you want to run this yourself OOC, pull the changes and run `tsc && npm start` from either service directory.
- Corollary to the above: I'll change tslint to eslint in a separate PR (tslint is deprecated, but most public how-to documentation hasn't been updated).
- For the directory structure, I've roughly used:

```
-- healthmap-gdo-temp
---- $workstreamName
------ $serviceWithinWorkStream
-------- service-specific files: code, .gitignore, so forth
---- $workstreamName
<...>
```

### CODE REVIEW

**Comment by axmb** [comment]
[Create baseline Node/TS curation server](https://trello.com/c/fQzatHpv/45-create-baseline-node-ts-curation-server)

**Comment by axmb** [comment]
[Create baseline Node/TS data server](https://trello.com/c/o77GfkzW/44-create-baseline-node-ts-data-server)

File: verification/curator-service/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "start": "node dist/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (khmoran) [inline verification/curator-service/package.json:12]: other repo is using MIT -- should we use that here?

Comment (axmb) [reply verification/curator-service/package.json]: Yep! The license file in this repo is MIT, too. Thanks for the catch.

**Pull request merged**